aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -517,8 +599,15 @@ public class AwsProperties implements Serializable {
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
   private String httpClientType;
+  private Long httpClientApacheConnectionAcquisitionTimeoutMS;
+  private Long httpClientApacheConnectionMaxIdleTimeMS;
```

Comment:
[nit] either let's make everything suffix as `MS` or `Ms`.

Reply:
```suggestion
  private Long httpClientApacheConnectionAcquisitionTimeoutMs;
  private Long httpClientApacheConnectionMaxIdleTimeMs;
```

Reply:
Thank you for your suggestions. I changed all relevant suffixes to `Ms`